migration-guide.md

```diff
@@ -189,14 +189,14 @@ riot.install(function(component) {
 ```
 
 Now your code and your helpers will be completely customizable without relying on the framework built in.
-This means that you will have more freedom and less bugs coming from your third party code...and don't forget to remember that:
+This means that you will have more freedom and fewer bugs coming from your third party code...and don't forget to remember that:
 
 > ...Explicit is better than implicit...
 
 
 ### Observable
 
-The Observable pattern was completely integrated into the previous Riot.js versions. This was an opinionated decision that might not work for all users. Riot.js 3 leaves you the decision regarding which programming pattern to use in your application and for this reason the observable helpers were completely removed from the source code in favor of a more generic approach.
+The Observable pattern was completely integrated into the previous Riot.js versions. This was an opinionated decision that might not work for all users. Riot.js 4 lets you decide which programming pattern to use in your application and for this reason the observable helpers were completely removed from the source code in favor of a more generic approach.
```

Comment:
I think this meant to say Riot 4 instead of Riot 3, but not completely sure. Let me know if this was intentional.